pkg/fanal/analyzer/sbom/sbom_test.go

```diff
@@ -0,0 +1,69 @@
+package sbom
+
+import (
+	"context"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+)
+
+func Test_sbomAnalyzer_Analyze(t *testing.T) {
+	tests := []struct {
+		name    string
+		file    string
+		want    *analyzer.AnalysisResult
+		wantErr require.ErrorAssertionFunc
+	}{
+		{
+			name: "valid spdx file",
+			file: "testdata/spdx.json",
```

Comment:
Maybe we need to add testcase for cyclonedx or are testcases from `sbom` package enough?
wdyt?

Reply:
Added
https://github.com/aquasecurity/trivy/pull/4210/commits/c6c2727cee57448abcb4ef823be9ddc642ec3343